### ISSUE TITLE
I read your article Radio Wave Classifier in Python

### ISSUE THREAD

**Issue by skywalkers2012** [lukerbs/gsi-internship#3]
I read your article [Radio Wave Classifier in Python](https://medium.com/gsi-technology/residual-neural-networks-in-python-1796a57c2d7) 
I download link to the [weights](https://drive.google.com/open?id=1Ej-W_IdkTv6qXBh6pxVXJ1dgPfukcDik) for the model that I your trained
I load the provided weights and try testing out some signal examples
Use the test data you provided singals_dataset / test / signals.npy
but the accuracy: 60.66%

Part of the code is as follows
model=load_model('weights.hdf5')
model.summary();
x_test = np.load(path + 'signals.npy')
y_test = np.load(path + 'labels.npy')
score = model.evaluate(x_test, y_test, batch_size=1024)
print("%s: %.2f%%" % (model.metrics_names[1], score[1]*100))

Is there something wrong with me?

**Comment by shaik1610** [comment]
Yes....The maximum accuracy even by training the model is 61%.I trained my model using HPC environment rather than a GPU as mentioned in the blog.In the blog notice that maximum classification accuracy is 60% but testing on high* SNR dataset gives you accuracy >90%.
